### ISSUE TITLE
Possible issue on Polish zloty?

### ISSUE THREAD

**Issue by cgbahk** [quantmind/ccy#39]
First, I'm not Polish :) Just found this while comparing with other references.

https://github.com/quantmind/ccy/blob/a510f2bcc97641df30b0afb2d6f913b5d1c5f64c/ccy/core/data.py#L442

I think it needs to be
```
        symbol_raw=r"\u007a\u0142",
```

```
>>> print("\u0050\u0142")
Pł
>>> print("\u007a\u0142")
zł
```

Reference: https://en.wikipedia.org/wiki/Polish_z%C5%82oty

I can find no google search result of `Pł` which is related with any currency.

**Comment by lsbardel** [comment]
thanks, can you create a PR?

**Comment by lsbardel** [comment]
Fixed in v1.4.0
